src/control/server/faults.go

```diff
@@ -110,8 +111,8 @@ func FaultPoolInvalidRanks(invalid []ranklist.Rank) *fault.Fault {
 
 	return serverFault(
 		code.ServerPoolInvalidRanks,
-		fmt.Sprintf("pool request contains invalid ranks: %s", strings.Join(rs, ",")),
-		"retry the request with a valid set of ranks",
+		fmt.Sprintf("pool request contains ranks with invalid state: %s", strings.Join(rs, ",")),
+		"retry the request with a set of ranks that have valid states",
```

Comment:
Feels a little weird and potentially confusing to lump in "bad state" (is just just excluded ranks?) with "nonexistent."

Reply:
agreed , will fix